content/swiftui/concepts/views/terms/picker/picker.md

```diff
@@ -0,0 +1,71 @@
+---
+Title: 'Picker'
+Description: 'A control that selects a value from a set of exclusive values.'
+Subjects:
+  - 'Mobile Development'
+  - 'Computer Science'
+Tags:
+  - 'SwiftUI'
+  - 'SwiftUI Views'
+  - 'Views'
+CatalogContent:
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+A `Picker` allows user to select a specific data between a set of data and can be displayed in different styles based on the type of data and its ultimate use.
```

Comment:
```suggestion
A `Picker` allows a user to select a specific data value between a set of data. It can be displayed with different styles based on the type of data and its ultimate use.
```